### PR TITLE
WIP: Add the horizon serverspec

### PR DESCRIPTION
**Pull request by vlope093**
Dependent on PR #1554

### CODE REVIEW

File: roles/horizon/templates/etc/serverspec/horizon_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+files = ['access.log', 'error.log', 'other_vhosts_access.log']
+files.each do |file|
+  describe file("/var/log/apache2/#{file}") do
+    it { should exist }
+    it { should be_owned_by 'root'}
+  end
+end
+
+describe file('/etc/openstack-dashboard/local_settings.py') do
+  it {should contain 'DEBUG = False'}
```
Comment (epwn) [inline roles/horizon/templates/etc/serverspec/horizon_spec.rb]: There's some more inconsistent whitespace on this line @vlope093

File: roles/horizon/tasks/serverspec.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+- name: serverspec checks for keystone
```
Comment (epwn) [inline roles/horizon/tasks/serverspec.yml]: should be "serverspec checks for horizon"

**Comment by j2sol** [comment]
ok to test

**Comment by j2sol** [comment]
retest

**Comment by j2sol** [comment]
retest

**Pull request merged**